telegram_bot.py

```diff
@@ -0,0 +1,44 @@
+import requests  # type: ignore
+from typing import Optional
+from telegram import Update
+from telegram.ext import (
+    filters,
+    ApplicationBuilder,
+    CallbackContext,
+    CommandHandler,
+    MessageHandler,
+)
+
+
+class Bot:
+    def __init__(self, api_key: Optional[str], cookies: str, url: str):
```

Comment:
I'm not entirely sure to pass the cookies and the F1Fantasy URL to the Bot constructor, we could pass them to the methods which actually need them. But we could refactor this later. I added this comment as a reminder.

If you are interested, check the next comment to see how we could handle those parameters